stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1437,6 +1437,13 @@ function axpby!(α, x::AbstractArray, β, y::AbstractArray)
     y
 end
 
+function scal!(α, x::AbstractArray)
```

Comment:
This needs a method for spare arrays that iterates over nonzeros.

Reply:
Almost no other methods in this file has support for sparse arrays. It's kind of strange to have support for that in the `BLAS` module.